### ISSUE TITLE
Forms / iOS: Build error after updating to beta 23

### ISSUE THREAD

**Issue by Sebastian1989101** [Mapsui/Mapsui#642]
After updating to Mapsui 2.0 beta 23, I get the following error message if I try to build the project: 

`Error MT2101: Can't resolve the reference 'SkiaSharp.SKTypeface SkiaSharp.SKTypeface::FromFamilyName(System.String)', referenced from the method 'System.Void Mapsui.Rendering.Skia.LabelRenderer::UpdatePaint(Mapsui.Styles.LabelStyle,System.Single)' in 'SkiaSharp, Version=1.68.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756'. (MT2101)`

**Comment by pauldendulk** [comment]
I don't get the build error. You are building on Mac? 

SkiaSharp.Svg has no 1.68 release. That might have something to do with this.

**Comment by pauldendulk** [comment]
https://github.com/mono/SkiaSharp.Extended/issues/68#issuecomment-462200821

**Comment by pauldendulk** [comment]
You should your update your own SkiaSharp packages to 1.68.

**Comment by pauldendulk** [comment]
> the project

Is that Mapsui or your app?

**Comment by charlenni** [comment]
Am I missed something? This error has nothing to do with the SVG package. It has to do with the SKTypeface implementation.

So 2.0 beta 22 worked and beta 23 not. Correct? If I‘m right, than @pauldendulk changed the Skia Package already in beta 22. Difference between both are only a small change in Mapsui.Forms.

I would say, that we should search the problem elsewhere. Perhaps it is really a problem with Skia package?

**Comment by pauldendulk** [comment]
The SVG package references an older version of SkiaSharp. Updating might cause mixing up version. I do not actually think this is the problem but when investigating a bug I try to generate several hypotheses.

**Comment by pauldendulk** [comment]
SkiaSharp was updated in beta 23. So, it is undoubtedly related to the update. The question is how.
https://github.com/Mapsui/Mapsui/releases

**Comment by pauldendulk** [comment]
I have no updated the nuspec. That is probably it.

**Comment by charlenni** [comment]
Sorry, I thought the update to 1.68.0 was bevor beta 22.

There are changes in SkiaSharp, because Skia removes font handling. They say, that this isn‘t a Skia problem. So SkiaSharp try to mimik the old behavior. Perhaps the problem is because of this?

**Comment by pauldendulk** [comment]
Beta 24 was released with updated nuget dependencies. Let's see if this fixed the problem.

**Comment by Sebastian1989101** [comment]
Yes, I built my iOS project on a Mac through a TFS build agent which also do the upload to ITC for me. Except for Mapsui, my app runs without SkiaSharp. Depending on the file changes when I updated the Mapsui package, I notice this one here in the iOS config file: 

```
      <dependentAssembly>
        <assemblyIdentity name="SkiaSharp" publicKeyToken="0738eb9f132ed756" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-1.60.0.0" newVersion="1.60.0.0" />
      </dependentAssembly>
      <dependentAssembly>
        <assemblyIdentity name="SkiaSharp.Views.iOS" publicKeyToken="0738eb9f132ed756" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-1.60.0.0" newVersion="1.60.0.0" />
      </dependentAssembly>
```

Maybe this causes the problem? I will try the latest beta release later today.

**Comment by pauldendulk** [comment]
beta 25 was released.

**Comment by pauldendulk** [comment]
This was an error in the nuspec which was fixed in 2.0.0-beta.25
